### PR TITLE
Add new icons for Xfce Desktop Environment

### LINKED ISSUE #285: Icon Request : Create archive
screenshot_2018-01-12_19-02-28

Im sorry...i couldnt find the exact location of the image,so i just edited it out from a context menu..

here it is...
screenshot_2018-01-12_19-08-16

thank u soooo much for making these icons..they are beautiful as hell...

### LINKED ISSUE #414: Missing Xfce menu icons
### Application information

| Application name | Icon name  | Context     | Icon reference |
| ---------------- | :--------- | :---------- | :------------- |
| Mail Reader    | `emblem-mail` | Application | 			   |

### Desktop file

```
[Desktop Entry]
Version=1.0
Type=Application
Exec=exo-open --launch MailReader %u
Icon=emblem-mail
StartupNotify=true
Terminal=false
Categories=Network;X-XFCE;X-Xfce-Toplevel;
OnlyShowIn=XFCE;
X-XFCE-MimeType=x-scheme-handler/mailto;
X-AppStream-Ignore=True
Name=Mail Reader
Comment=Read your email
```

image

### PR DESCRIPTION
**Pull request by sptndc**
### Description

Add icon `emblem-mail.svg`, `stock_xfburn.svg`, `tap-create.svg`

### CODE REVIEW

**Comment by keeferrourke** [comment]
`nm-secure-lock` is under `symbolic/`? should it not be called `nm-secure-lock-symbolic` then? iirc there is already a nm-secure-lock

**Comment by keeferrourke** [comment]
Otherwise this looks good :)

**Comment by sptndc** [comment]
Hello @keeferrourke, please tell me where `nm-secure-lock` icon is located.

**Comment by keeferrourke** [comment]
Hm my bad! It looks like it doesn't exist. I believe it *should* be located under the `status` directory though.

**Comment by sptndc** [comment]
Okay, I revert a commit to fix issue first.

**Comment by keeferrourke** [comment]
Last criticism: `emblem-mail` should belong to the `emblems/` directory.

**Comment by sptndc** [comment]
Okay, I'll fix it later.

File: emblems/scalable/emblem-mail.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....5 0 12.979 9.178 23.5 20.5 23.5s20.5-10.521 20.5-23.5c0-1.189-.079-2.357-.228-3.5h27.456c-.149 1.143-.228 2.311-.228 3.5 0 12.979 9.178 23.5 20.5 23.5s20.5-10.521 20.5-23.5c0-1.189-.079-2.357-.228-3.5h23.344c-.075.821-.116 1.655-.116 2.5 0 12.979 9.178 23.5 20.5 23.5s20.5-10.521 20.5-23.5c0-.845-.041-1.679-.116-2.5h27.232c-.075.821-.116 1.655-.116 2.5 0 12.979 9.178 23.5 20.5 23.5s20.5-10.521 20.5-23.5c0-.845-.041-1.679-.116-2.5h26.116v27.116c-11.803 1.09-21 9.799-21 20.384s9.197 19.294 21 20.384v27.232c-11.803 1.09-21 9.799-21 20.384s9.197 19.294 21 20.384v24.232c-11.803 1.09-21 9.799-21 20.384s9.197 19.294 21 20.384v27.232c-11.803 1.09-21 9.799-21 20.384s9.197 19.294 21 20.384v23.233c-11.803 1.089-21 9.798-21 20.383s9.197 19.294 21 20.383v27.234c-11.803 1.089-21 9.798-21 20.383s9.197 19.294 21 20.383v25.497c-10.825 1.831-19 10.141-19 20.12" fill="url(#b)" transform="matrix(.093963726517 -.010745682911 -.008793144971 -.077123327814 5.437642817412 49.813012073681)"/><rect fill="url(#a)" height="36.075733" ry=".531829" transform="matrix(.985560999529 -.112708706223 .11237220789 .985599416862 -438.011826217655 -445.734705829515)" width="31.074825" x="393.59326" y="504.85712"/></svg>
```
Comment (keeferrourke) [inline emblems/scalable/emblem-mail.svg]: This file needs to be tidied.

**Pull request merged**